apps/cowswap-frontend/.env

```
@@ -148,3 +148,4 @@
 # CROWDIN_PERSONAL_TOKEN
 
 REACT_APP_COW_CDN=https://files.cow.fi
+ENABLE_EXPERIMENTAL_COREPACK=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENABLE_EXPERIMENTAL_COREPACK key should go before the REACT_APP_COW_CDN key

(UnorderedKey)